<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="11" hostname="DESKTOP-DJ4FDME" failures="5" tests="5" name="Test(failed)" time="0.174" errors="0" timestamp="2024-11-21T21:31:46 IST">
  <testcase classname="demoTestNG.SecnodScenatio" name="fifth_testcase" time="0.039">
    <failure type="java.lang.AssertionError" message="expected [Hello wprld] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello wprld] but found [Hello]
at demoTestNG.SecnodScenatio.fifth_testcase(SecnodScenatio.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.SecnodScenatio" name="forth_testcase" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [Hello wprld] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello wprld] but found [Hello]
at demoTestNG.SecnodScenatio.forth_testcase(SecnodScenatio.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.ThirdScenatio" name="Third_testcase" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [Hello wprld] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello wprld] but found [Hello]
at demoTestNG.ThirdScenatio.Third_testcase(ThirdScenatio.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Third_testcase -->
  <testcase classname="demoTestNG.ThirdScenatio" name="fifth_testcase" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [Hello wprld] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello wprld] but found [Hello]
at demoTestNG.ThirdScenatio.fifth_testcase(ThirdScenatio.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="first_testcase" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [Hello wprld] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello wprld] but found [Hello]
at demoTestNG.TestScenario.first_testcase(TestScenario.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.SecnodScenatio" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.ThirdScenatio" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.ThirdScenatio" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.ThirdScenatio" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.SecnodScenatio" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.SecnodScenatio" name="Third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- Third_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
</testsuite> <!-- Test(failed) -->
